pkg/util/rbac.go

```diff
@@ -2,6 +2,7 @@ package util
 
 import (
 	"context"
+	"k8s.io/utils/strings/slices"
```

Comment:
Please format this

Reply:
ok i fixed it
https://github.com/karmada-io/karmada/pull/2911/commits/f68c2a94131dbe766d52293d6957b387c737416e